test-cases/performance/perf-regression-latency-1TB-ebs-volumes.yaml

```diff
@@ -0,0 +1,35 @@
+test_duration: 600
```

Comment:
Since the 1TB takes too long, we can reduce it to 500GB.

Reply:
set 500GB